docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3.8'
+services:
+  catalog:
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    container_name: sd-catalog-backend
+    image: sd-catalog-backend:latest
+    restart: unless-stopped
+    environment:
+      MONGODB_DATABASE: "sda_catalog"
+      MONGODB_HOSTNAME: mongodb
```

Comment:
Can you please add these 2 environment variables for the `sd-catalog-backend` container? 
     ```
 MONGODB_USERNAME: admin
 MONGODB_PASSWORD: admin
```
Otherwise the `sd-catalog-backend` container is failing with the following errors:
```
sd-catalog-backend |   File "/usr/local/lib/python3.9/os.py", line 679, in __getitem__
sd-catalog-backend |     raise KeyError(key) from None
sd-catalog-backend | KeyError: 'MONGODB_USERNAME'
```

Reply:
Implemented the recommendation